Vagrantfile

```diff
@@ -46,6 +46,8 @@ Vagrant.configure(2) do |config|
     echo "Creating a virtualenv and installing requirements!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
     echo "================================================================================"
     mkvirtualenv alliance
+    workon alliance
+    cd /vagrant
```

Comment:
If I am correct, this is looking at the present working directory (presumably where the code lives). Therefore one could assume:

| Host | VM |
|---|---|
|`<path_to_repo>`|`/vagrant`|
|`<path_to_repo>/alliance/manage.py`|`/vagrant/alliance/manage.py`|
Correct?